.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules/
-.dist/
+.dist/*
```

Comment:
We want the folder, just not its file, as it's referenced in the Readme as already existing.